lib/src/Pos/Client/CLI/Params.hs

```diff
@@ -38,7 +38,7 @@ loggingParams defaultName CommonNodeArgs{..} =
     { lpHandlerPrefix = logPrefix commonArgs
     , lpConfigPath    = logConfig commonArgs
     , lpDefaultName   = defaultName
-    , lpConsoleLog    = Nothing -- no override by default
+    , lpConsoleLog    = Just (not $ logConsoleOff commonArgs)
```

Comment:
I think there's a subtle difference between `Nothing` and `Just False`

```hs
    overrideConsoleLog :: LoggerConfig -> LoggerConfig
    overrideConsoleLog = case lpConsoleLog of
        Nothing    -> identity
        Just True  -> (<> defaultInteractiveConfiguration Info)
                      -- add output to the console with severity filter >= Info
        Just False -> lcLoggerTree . ltHandlers %~ filter (not . isWritingToConsole)
```

So now, if we don't provide the flag, it will necessarily add a console logger :thinking: 
Is it possible to capture that in the parser (and have this three-way choice instead of a raw boolean?)

Reply:
Thank for your comment, your input is kept in CBR-473.
There will be more requirements gathering.